### PR TITLE
Update metadata templates

### PR DESCRIPTION
**Pull request by hjkristenson**
This PR updates the metadata templates to address the following: 

- add reference links for the MCF algorithm
- clarify that the sign convention for the unwrapped phase is opposite to that used for the displacement maps
- add references to new HyP3 citation

### CODE REVIEW

File: hyp3_gamma/metadata/templates/insar/readme.md.txt.j2 [hunk 1]
```diff
@@ -52,6 +52,10 @@ When using this data in a manuscript and/or crediting datasets used for analysis
 
     ASF DAAC HyP3 {{ processing_date.year }} using the {{ plugin_name }} plugin version {{ plugin_version }} running {{ processor_name }} release {{ processor_version }}. Contains modified Copernicus Sentinel data {{ secondary_granule_name[17:21] }}, processed by ESA.
 
+To cite HyP3 software in manuscripts, use the citation available at https://github.com/ASFHyP3/hyp3-docs:
```
Comment (jhkennedy) [inline hyp3_gamma/metadata/templates/insar/readme.md.txt.j2:55]:
```suggestion
To cite HyP3 in manuscripts, cite our documentation available at https://github.com/ASFHyP3/hyp3-docs:
```

I think this citation is intended more as "all" of HyP3 (the concept, all the software working together, etc.)

File: hyp3_gamma/metadata/templates/rtc/readme.md.txt.j2 [hunk 2]
```diff
@@ -44,6 +44,10 @@ When using this data in a manuscript and/or crediting datasets used for analysis
 
     ASF DAAC HyP3 {{ processing_date.year }} using the {{ plugin_name }} plugin version {{ plugin_version }} running {{ processor_name }} release {{ processor_version }}. Contains modified Copernicus Sentinel data {{ granule_name[17:21] }}, processed by ESA.
 
+To cite HyP3 software in manuscripts, use the citation available at https://github.com/ASFHyP3/hyp3-docs:
```
Comment (jhkennedy) [inline hyp3_gamma/metadata/templates/rtc/readme.md.txt.j2:47]: ditto here

File: hyp3_gamma/metadata/templates/insar/readme.md.txt.j2 (hunk 1 above)
Comment (hjkristenson) [reply hyp3_gamma/metadata/templates/insar/readme.md.txt.j2]: @jhkennedy I was reading this again now that it's in situ, it reads a little funny with cite twice. Any thoughts on alternative wording that would still convey the right meaning? The best I could come up with was: 
"To reference HyP3 in manuscripts, cite our documentation available at https://github.com/ASFHyP3/hyp3-docs:"

Comment (jhkennedy) [reply hyp3_gamma/metadata/templates/insar/readme.md.txt.j2]: @hjkristenson totally agree -- I struggled with the same. I like:

> To reference HyP3 in manuscripts, cite our documentation available at https://github.com/ASFHyP3/hyp3-docs:

**Review by jhkennedy — APPROVED** [review]

**Pull request merged**